homeassistant/components/nina/const.py

```diff
@@ -24,6 +24,8 @@
 ATTR_START: str = "start"
 ATTR_EXPIRES: str = "expires"
 
+CONST_WARNING_ACTIVE: str = "active"
```

Comment:
A better name for this might be `ATTR_ACTIVE` given how it's used.  In `warn_obj` every other property key is listed as `ATTR_*` and this appears to be used the same way so should it match?

Reply:
The `ATTR_*` fields are displayed one to one as entity attribute, `CONST_WARNING_ACTIVE` is only for internal use to provide the status of the warning. So I think the name is ok.

Reply:
In any case, we don't have to prefix is with `CONST_`.

Ideally, it should not return a dict, but e.g., a dataclass instance.